bench/perf/index.js

```diff
@@ -7,6 +7,8 @@ import * as parse5 from '../../packages/parse5/dist/index.js';
 import { ParserStream as parse5Stream } from '../../packages/parse5-parser-stream/dist/index.js';
 import * as parse5Upstream from 'parse5';
 
+/* eslint-disable no-console */
```

Comment:
I don’t see a good reason to put file based ignores after import (export?) and before global definitions. I’d say before the `import`s makes more sense?

Reply:
Sure I just threw it somewhere near the top

Happy to move it

Reply:
turns out it was like that before and I had just left it as-is

I've moved it now, though